abstractions/ruby/microsoft_kiota_abstractions/lib/microsoft_kiota_abstractions/authentication/base_bearer_token_authentication_provider.rb

```diff
@@ -1,27 +1,28 @@
+# frozen_string_literal: true
+
 require 'concurrent'
+require_relative './authentication_provider'
+require_relative './access_token_provider'
 
 module MicrosoftKiotaAbstractions
- class BaseBearerTokenAuthenticationProvider
-    include MicrosoftKiotaAbstractions::AuthenticationProvider
+  # Provides a base class for implementing AuthenticationProvider for Bearer token scheme
+  class BaseBearerTokenAuthenticationProvider
```

Comment:
shouldn't this be kept? (might be my ruby ignorance)
include MicrosoftKiotaAbstractions::AuthenticationProvider